### PR TITLE
Add Symfony 5.0+ compatibility

### PR DESCRIPTION
**Pull request by tomsihap**
Updated minimum PHP version to 7.2.5 and Symfony ^5.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Console-Selection
 
+This project is forked from eddiriarte/console-select to make console-select work with Symfony 5.0+.
```
Comment (eddiriarte) [inline README.md]: @tomsihap this line can be removed, right? ;)

File: composer.json [hunk 2]
```diff
@@ -1,18 +1,23 @@
 {
-    "name": "eddiriarte/console-select",
+    "name": "tomsihap/console-select",
```
Comment (eddiriarte) [inline composer.json]: please set old name back

**Review by eddiriarte — COMMENTED** [review]
Besides of the fork notification in the README file and incorrect name in composer.json, looks good to me...

After merge I would release the lib as v3.0.0 since support for older versions is beeing dropped by requiring PHP7.2+ and Symfony ^5

**Review by tomsihap — COMMENTED** [review]
Fixing composer.json and Readme comments.

**Review by eiriarte-mendez — APPROVED** [review]
👍 thx for support

**Review by eddiriarte — APPROVED** [review]

**Comment by tomsihap** [comment]
Thanks for the approval. One of my first PR so thanks for the guidance, it allowed me to tinker a little bit to learn ! :slightly_smiling_face:

**Pull request merged**